api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -80,16 +79,17 @@ public MailServiceImpl(
   }
 
   @Override
-  public void sendWelcomeEmail(final String contactEmail, final String password, final User user)
+  public void sendWelcomeEmail(
+      final String contactEmail, final String password, final String username)
       throws MessagingException {
 
     final String htmlMessage =
-        buildHtml(WELCOME_RESOURCE, welcomeMessageSubstitutionMap(password, user));
+        buildHtml(WELCOME_RESOURCE, welcomeMessageSubstitutionMap(password, username));
 
     sendWithRetries(
         Collections.singletonList(contactEmail),
         "Your new All of Us Researcher Workbench Account",
-        String.format("Welcome for %s", user.getName()),
+        String.format("Welcome for %s", username),
```

Comment:
This is a change but not a user-facing one.  We now log "Welcome for janeX99@researchallofus" instead of "Welcome for Jane Xavier" - which is more precise when humans have multiple users (like I do in Test)